Client/Frontend/LoginManagement/BreachAlertsDetailView.swift

```diff
@@ -211,3 +208,10 @@ class BreachAlertsDetailView: UIView {
         }
     }
 }
+
+// MARK: - ThemeApplicable
+extension BreachAlertsDetailView: ThemeApplicable {
+    func applyTheme(theme: Theme) {
+        backgroundColor = theme.colors.iconWarning
+    }
+}
```

Comment:
I believe the team would like to move off using extensions where possible. Instead, I would add `ThemeApplicable` to line 8 and add `applyTheme` towards the bottom of the class.

Reply:
We have a wiki page up for the view on extensions as well, if you'd like to read. https://github.com/mozilla-mobile/firefox-ios/wiki/When-To-Use-Extensions